tracer_test.go

```diff
@@ -255,6 +255,11 @@ func TestEmptySpanContextAsParent(t *testing.T) {
 	assert.True(t, ctx.IsValid())
 }
 
+func TestTracerClose(t *testing.T) {
+	_, tc := NewTracer("x", NewRemotelyControlledSampler("svc"), NewNullReporter())
+	assert.NoError(t, tc.Close())
```

Comment:
it doesn't actually test that remote sampler was actually stopped. I.e. if you don't return error from sampler's Close() (thus not implement io.Closer) your test would succeed just the same.